### PR TITLE
Landing page header

### PR DESCRIPTION
**Pull request by nikballal**
pixel-perfect couldnt be achieved.

### CODE REVIEW

**Comment by nikballal** [comment]
-  assigned routes in expression format in Navbar.js file, changed it to string format due to href error

File: pages/index.jsx [hunk 1]
```diff
@@ -12,6 +15,41 @@ const Header = styled.header`
   background: white;
 `;
 
+const Container = styled.div`
+  width: 1280px;
```
Comment (jkettmann) [inline pages/index.jsx:19]: To make this responsive you can use

```
width: 100%;
max-width: 1280px;
```

File: pages/index.jsx [hunk 2]
```diff
@@ -12,6 +15,41 @@ const Header = styled.header`
...
+  height: 80px;
+  padding: 0 2rem;
+  display: flex;
+  justify-content: space-between;
```
Comment (jkettmann) [inline pages/index.jsx:23]: Interestingly this was enough when I built the header to make it pixel-perfect (which I didn't expect actually). No idea what the difference is in your code. But one way to make this close to pixel-perfect is to wrap the logo and the link in a `div` and set `flex: 1` on those divs. This way they expand to take up equal space and the nav links are positioned in the middle.

image

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-2)

**Pull request merged**